pynumdiff/tests/test_linear_model.py

```diff
@@ -26,76 +25,75 @@
 dt = 0.01
 
 
-class TestLM(TestCase):
```

Comment:
I didn't even know that pytest can run unittest TestCases, but evidently it can. There's no need to have dependency on both, though. It's simpler and fewer lines to do tests the pytest way.